playwright-tests/tests/page-objects/trace-explorer-view.ts

```diff
@@ -0,0 +1,42 @@
+/********************************************************************************
+ * Copyright (C) 2024 Ericsson and others.
```

Comment:
Shouldn't be the copyright Ericsson and others? I think you wrote this file.

2024?

Reply:
Yes, this is an original file. In short, I have taken since several years, to add the "and others" part preemptively when creating a new file. It's easy to miss updating this later, when another entity contributes to the file, resulting potentially in it appearing like the original copyright holder is taking credit for contributions of another. Until such other contribution happens, the git history (log) will confirm that the original copyright holder is the sole owner, and that the "others" set is empty. This way, the copyright line needs never be updated. 

TL;DR: IMHO, it's not wrong to add "and others" since it's easily demonstrated and there are no others, but it potentially becomes wrong not to have the mention (i.e. to forget to add it), when there demonstrably is at least one "other".